### ISSUE TITLE
Chose margins direction and mention it in README

### ISSUE THREAD

**Issue by mr-mig** [brainly/style-guide#233]
We need to chose a single direction for vertical and horizontal margins.
This choice should be stated in README.

See this article: http://csswizardry.com/2012/06/single-direction-margin-declarations/

Related to #13

**Comment by aju** [comment]
bottom/right ?:)

**Comment by vergilius** [comment]
@aju I agree with the bottom for sure, but how are we going to handle item that should have margin/padding on left side and its first item? `first-child` and `margin-left`?

**Comment by aju** [comment]
@vergilius yeah actually thats a good solution:) other than that we usually align items to left so setting right margin have more sense for me.

**Comment by vergilius** [comment]
We should also decide how paddings/margins should be applied to 2 components ie one is inside another

**Comment by mr-mig** [comment]
The problems I faced so far:
* Parent-child paddings/margins
* margins for wrappable elements in a row
* margins and paddings for container *with borders* and VR applied

I've tried different solutions:
1. Padding for parent - does not play with wrapping holes at all
2. Padding (left-top) for parent + margin (bottom-right) for child - does not play nice with borders
3. Padding (left-top) for parent (1/2 rhythm) + margin (bottom-right) for child (1/2 rhythm) - worked out ok for now, but need more testing
